library/src/main/res/values/library_photoview_strings.xml

```diff
@@ -14,8 +14,10 @@
 	<string name="library_photoview_isOpenSource">true</string>
 	<string name="library_photoview_repositoryLink">https://github.com/chrisbanes/PhotoView</string>
 	<!-- ClassPath for autoDetect section -->
-	<string name="library_photoview_classPath">com.github.chrisbanes.PhotoView</string>
+	<string name="library_photoview_classPath">uk.co.senab.photoview.BuildConfig</string>
```

Comment:
Ah, you're right - sorry...this is a different PhotoView.  I should have checked more carefully.  I'll add a new file for this.